renderdoc/driver/vulkan/wrappers/vk_resource_funcs.cpp

```diff
@@ -1620,6 +1618,39 @@ VkResult WrappedVulkan::vkBindImageMemory(VkDevice device, VkImage image, VkDevi
         state->isMemoryBound = true;
     }
 
+    VkResourceRecord *record = GetRecord(image);
+
+    if(record->resInfo->imageInfo.isAHB)
```

Comment:
This handling also needs to be replicated in `WrappedVulkan::vkBindImageMemory2`.